databricks/koalas/base.py

```diff
@@ -224,6 +289,16 @@ def __rfloordiv__(self, other):
     __rand__ = _column_op(spark.Column.__rand__)
     __ror__ = _column_op(spark.Column.__ror__)
 
+    # NDArray Compat
+    def __array_ufunc__(self, ufunc: Callable, method: str, *inputs: Any, **kwargs: Any):
+        result = maybe_dispatch_ufunc_to_dunder_op(
+            self, ufunc, method, *inputs, **kwargs)
+        if result is not NotImplemented:
+            return result
+        else:
+            # TODO: support more APIs?
```

Comment:
Maybe we can delegate to pandas UDF?

Reply:
Oh that's nice suggestion. Let me investigate a bit more about this. It will only work when the output is n to n but I'm sure there will be the case.

Reply:
Yeah, if we can delegate to pandas UDF in a general way, we can take time to add more Spark native functions like `np.sqrt` or `np.log`.